docs/book/src/forc/plugins/forc_client/forc_call.md

```diff
@@ -24,13 +22,35 @@ impl ContractABI for Contract {
 }
 ```
 
+### List callable functions of a contract given it's ABI file
+
+```bash
+forc call 0xe18de7c7c8c61a1c706dccb3533caa00ba5c11b5230da4428582abf1b6831b4d \
+  --abi ./out/debug/counter-contract-abi.json \
+  --list-functions
+```
+
+Output:
+
+```log
+Available functions in contract: 0xe18de7c7c8c61a1c706dccb3533caa00ba5c11b5230da4428582abf1b6831b4d
+
+add(a: u64, b: u64) -> u64
+  forc call \
+    --abi ./out/debug/counter-contract-abi.json \
+    0xe18de7c7c8c61a1c706dccb3533caa00ba5c11b5230da4428582abf1b6831b4d \
+    add "0" "0"
```

Comment:
nit: It would be nice to omit the quotes when they aren't necessary

Reply:
https://github.com/FuelLabs/sway/pull/7018/commits/7141b5c09c76f939c52f8656b3f5928162bf34f3